k4e-agent-sti.sh

```diff
@@ -74,32 +128,65 @@ createrepo /var/www/html/k4e-repo/
 #-----------------------------------
 git clone https://github.com/ydayagi/r4e.git /home/builder/r4e
 cd /home/builder/r4e
-./r4e-image.sh $(hostname) edgedevice helios05.lab.eng.tlv2.redhat.com:8888 http://$(hostname)/k4e-repo/
+./r4e-image.sh -s $(hostname) -i $IMAGE_NAME -o $HTTP_API -p http://$(hostname)/k4e-repo/
 
 #----------------------------------------
 # Create ISO for image and serve via http
 #----------------------------------------
-cat << EOF > /tmp/empty.toml
-name = "empty"
-description = "Empty blueprint"
+ISO_BLUEPRINT=$(mktemp)
+cat << EOF > $ISO_BLUEPRINT
+name = "edgedevice-iso"
```

Comment:
what will happen if that script would be run again?

Reply:
iiuc, the edgedevice-iso blueprint will be updated.
however, the script itself isn't idempotent. there is some cleanup to allow re-run of it.

Reply:
I think we need to add a cleanup section or a cleanup script (maybe destroy)

Reply:
I'll update one of the stories about refactoring this repo, since none of the scripts can be run over and not sure if that will be needed.